src/resolver/nodeModuleLookup.ts

```diff
@@ -86,7 +88,7 @@ export function findTargetFolder(props: IResolverProps, parsed: IModuleParsed):
   if (localModuleRoot === undefined) {
     localModuleRoot = CACHED_LOCAL_MODULES[props.filePath] = findUp(props.filePath, "node_modules");
   }
-  if (!!localModuleRoot) {
+  if (!!localModuleRoot && paths.indexOf(localModuleRoot) === -1) {
```

Comment:
is it going to work on windows too? ;-)

Reply:
I think it will actually.  But it'd be better to be explicit about it.

I added this so it wouldn't check a path it already had checked before.  What's the correct way to handle windows paths for this?

Reply:
depends on what string values end up in there
path segment is seperated with  \ on Windows and / on LINUX/MAC

Reply:
Looked at not node path module, `node.sep` gives you the current on the system..


```js
const path = require('path')
const linux = `/home/project/node_modules`
const window = `\\home\\project\\node_modules`
console.log(linux.split(path.sep).join('/'))  //prints /home/project/node_modules
console.log(window.split(path.sep).join('/')) //prints /home/project/node_modules
```

Reply:
worst case scenario is that it will check the same directory twice.